crawl-ref/source/dat/des/arrival/simple.des

```diff
@@ -2508,3 +2508,44 @@ tt.........tt
  tt.......tt
   ttt@@@ttt
 ENDMAP
+
+NAME:   reono_arrival_forest_river
+TAGS:   arrival no_monster_gen no_trap_gen no_pool_fixup
+ORIENT: float
+SUBST:  w = w:5 W:1 t:2
+MAP
+ttttttttt
+twwwtwwwt
+twww{wwwt
+twwwWwwwt
+twwwWwwwt
+twwwWwwwt
+twwwWwwwt
+twwwWwwwt
+twwwWwwwt
+twwwWwwwt
+twwwWwwwt
+twwwWwwwt
+twww@wwwt
```

Comment:
The forest river vault is almost entirely water and has a single exit, which makes it quite hard to fight in. Ideally, entry vaults should give the player some tactical room to maneuver and/or multiple ways out of the vault in case they run into trouble immediately. To that end I would suggest adding a little bit more ground at the back to fight on and either a hatch downward or an additional pathway leading out in a different direction.